code/modules/uplink/uplink_items/ammunition.dm

```diff
@@ -35,7 +35,7 @@
 /datum/uplink_item/ammo/pistolap
 	name = "9mm Armour Piercing Magazine"
 	desc = "An additional 8-round 9mm magazine, compatible with the Makarov pistol. \
-			These rounds are less effective at injuring the target but penetrate protective gear."
+	These rounds are less effective at injuring the target but penetrate protective gear."
```

Comment:
These need to still be indented up ONE, just not two. They shouldn't align, just indicate something new.

Reply:
Should be done now.